src/rgw/rgw_admin.cc

```diff
@@ -7670,7 +7670,10 @@ int main(int argc, const char **argv)
 
     int ret = init_bucket(user.get(), tenant, bucket_name, bucket_id, &bucket);
     if (ret < 0) {
-      cerr << "ERROR: could not init bucket: " << cpp_strerror(-ret) << std::endl;
+      ldpp_dout(dpp(), 0) << "ERROR: " << __func__ << ": init_bucket " <<
+	cpp_strerror(-ret) << dendl;
+      cerr << "ERROR: could not init bucket: " << cpp_strerror(-ret) <<
+	std::endl;
```

Comment:
i think it should just be one or the other. log output is already being written to stderr by default, so we can probably just drop the `cerr`?

Reply:
My goal is to make it available in the _client.admin.*.log_ file. So I'll drop the cerr.

Reply:
Fixed.